### PR TITLE
build: remove wml dependency

### PR DESCRIPTION
**Pull request by erinleigh90**
#### Description of changes

Removes wml dependency and react-native development script. Updates CONTRIBUTING guide to direct developers to test ReactNative using Verdaccio.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by elorzafe — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -126,7 +125,6 @@
 		"webpack": "^5.75.0",
 		"webpack-bundle-analyzer": "^4.7.0",
 		"webpack-cli": "^5.0.0",
-		"wml": "0.0.83",
 		"winston": "3.11.0"
```
Comment (jimblanc) [inline package.json:128]: I think we can remove `winston` as well now, which was introduced here: https://github.com/aws-amplify/amplify-js/commit/768788d1d86135fad401f47ccef71b77d7048da2

**Review by kvramyasri7 — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -103,6 +103,8 @@ yarn run test --scope @aws-amplify/auth
 **Yarn Linking**
 The best way to develop locally and test is to link the individual package you’re working on and run lerna in watch mode.
 
+Note: to test using the react-native framework you will need to use [Verdaccio](#verdaccio)
```
Comment (kvramyasri7) [inline CONTRIBUTING.md:106]: just a nit u can ignore as well not requesting for changes 😅
```suggestion
Note: to test using the react-native framework you will have to use [Verdaccio](#verdaccio)
```

**Review by jimblanc — APPROVED** [review]

**Review by kvramyasri7 — DISMISSED** [review]
I am approving but if you need @Samaritan1011001  approval we can wait. But giving my approval

**Review by Samaritan1011001 — DISMISSED** [review]

**Review by cshfang — DISMISSED** [review]

**Review by jimblanc — APPROVED** [review]

**Review by ashika112 — APPROVED** [review]

**Review by kvramyasri7 — APPROVED** [review]
Thanks for putting this together Erin! ❤️

**Pull request merged**